packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -0,0 +1,277 @@
+import React from 'react';
+import styled from 'styled-components';
+import { GEL_GROUP_CD_MIN_WIDTH } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import {
+  C_WHITE,
+  C_POSTBOX,
+  C_SHADOW,
+  C_EBON,
+  C_METAL,
+} from '@bbc/psammead-styles/colours';
+import { string, oneOf, shape, node } from 'prop-types';
+import { getSansRegular, getSansBold } from '@bbc/psammead-styles/font-styles';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getPica, getLongPrimer } from '@bbc/gel-foundations/typography';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const BulletinWrapper = styled.div`
+  background-color: ${C_WHITE};
+  padding: ${GEL_SPACING};
+  position: relative;
+  text-decoration: none;
+
+  @media (min-width: ${GEL_GROUP_CD_MIN_WIDTH}rem) {
+    display: none;
+  }
+`;
+
+const IconWrapper = styled.span`
+  > svg {
+    color: ${C_WHITE};
+    fill: currentColor;
+    margin-right: ${GEL_SPACING};
+    height: ${GEL_SPACING_DBL};
+  }
+`;
+
+const PlayCTA = styled.div.attrs({ 'aria-hidden': true })`
+  background-color: ${({ isLive }) => (isLive ? C_POSTBOX : C_EBON)};
+  color: ${C_WHITE};
+  width: 100%;
+  display: block;
+  text-align: center;
+  padding: ${GEL_SPACING};
+  vertical-align: middle;
+`;
+
+const StyledSummary = styled.p`
+  ${({ script }) => script && getLongPrimer(script)};
+  ${({ service }) => getSansRegular(service)}
```

Comment:
We should check if `service` exists before calling it, same way we do with `script`. `${({ service }) => service && getSansRegular(service)}`